packages/passwordless-auth-construct/src/construct.ts

```diff
@@ -151,11 +159,21 @@ export class AmplifyPasswordlessAuth extends Construct {
         {
           defineAuthChallenge,
           createAuthChallenge,
-          verifyAuthChallengeResponse,
+          verifyAuthChallengeResponse: this.verifyAuthChallengeResponse,
         },
         props.magicLink
       );
     }
+
+    // Configure Sign Up without password
+    if (props.signUpNoPassword) {
+      new AmplifySignUpPasswordless(
```

Comment:
along with the prop rename, rename these related classes and names to `AmplifyPasswordlessSignUp`